code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -108,6 +109,7 @@
 			new /obj/item/grenade/plastic/c4 (src)
 			new /obj/item/grenade/plastic/x4 (src)
 			new /obj/item/grenade/plastic/x4 (src)
+			new /obj/item/card/emag(src)
```

Comment:
The emag seems like arbitrary powercreep, you already get plenty of breaching / B&E gear in this bundle.